src/Datadog.Trace/SpanContextPropagator.cs

```diff
@@ -86,6 +87,23 @@ public SpanContext Extract<T>(T headers)
             return new SpanContext(traceId, parentId, samplingPriority, null, origin);
         }
 
+        public IEnumerable<KeyValuePair<string, string>> ExtractHeaderTags<T>(T headers, IEnumerable<KeyValuePair<string, string>> headerTags)
+            where T : IHeadersCollection
+        {
+            Dictionary<string, string> tagsFromHeaders = new Dictionary<string, string>();
+
+            foreach (KeyValuePair<string, string> headerNameToTagName in headerTags)
+            {
+                string headerValue = ParseString(headers, headerNameToTagName.Key);
```

Comment:
I didn't notice before, but if a header contains multiple values, `ParseString()` only returns the one. Does the RFC for this feature define what to do with multiple values? Concatenating with commas is common, for example.

Reply:
I checked with the Java implementation and they take the first one, so this behavior aligns with that behavior